src/gui/plugins/visualize_lidar/VisualizeLidar.cc

```diff
@@ -150,18 +145,18 @@ void VisualizeLidar::LoadLidar()
     return;
 
   // assume there is only one engine loaded
```

Comment:
I realized that code around can be replaced by call `rendering::sceneFromFirstRenderEngine`